deploy/tools/envs-validator/schema.ts

```diff
@@ -103,12 +102,40 @@ const adButlerConfigSchema = yup
       .required(),
   });
 
+const adCustomConfigSchema = yup
+  .object()
+  .shape({
+    banners: yup
+      .array()
+      .of(
+        yup.object().shape({
```

Comment:
for type safety, it is better to put this schema in a separate variable and specify the type for it as `yup.ObjectSchema<AdCustomBannerConfig>` or add the type annotation for the whole `adCustomConfigSchema`

see `marketplaceAppSchema` as an example

Reply:
Okay